durablefunctionsmonitor.react/src/states/LoginState.ts

```diff
@@ -181,20 +187,20 @@ export class LoginState extends ErrorMessageState {
         })
 
         // Checking if it was a redirect from AAD
-        this._aadApp.handleRedirectCallback(() => { }, (authErr: Msal.AuthError, accountState: string) => {
 
-            console.log(`Failed to handle login redirect. name: ${authErr.name}, message: ${authErr.message}, errorCode: ${authErr.errorCode}, errorMessage: ${authErr.errorMessage}, accountState: ${accountState}`);
+        this._aadApp.handleRedirectPromise().catch(err => {
```

Comment:
As discovered, the promise returned by .handleRedirectPromise() actually needs to be _handled_. Immediatel upon redirect from AAD this._aadApp.getAllAccounts() would still be empty and will only get populated in that promise handler